core/services/pipeline/task.bridge.go

```diff
@@ -74,7 +74,7 @@ var _ Task = (*BridgeTask)(nil)
 
 var zeroURL = new(url.URL)
 
-const stalenessCap = time.Duration(30 * time.Minute)
```

Comment:
Should this have a comment explaining where it comes from or be wired through config?

Reply:
It does seem arbitrary. Introduced here: https://github.com/smartcontractkit/chainlink/pull/7964